.github/workflows/ci.yml

```diff
@@ -15,6 +15,7 @@ jobs:
   build-and-test:
     name: Run on ${{ matrix.os }} with SOFA ${{ matrix.sofa_branch }} and python ${{ matrix.python_version }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: true # as macos is always failing...
```

Comment:
This line should be in the `deploy` job, not the `build-and-test` job  🤐

Reply:
Indeed... but the `continue-on-error: true` is already there. 
So if I understand correctly, the Deploy artifacts was actually Ok, just not uploading the macOS release, other are properly uploaded, and we can close this PR.

Reply:
So why https://github.com/SofaDefrost/Cosserat/pull/82 is failing?

Reply:
Don't know yet. https://github.com/SofaDefrost/Cosserat/pull/82 fails because it fails to get the master-nightly release of SofaPython3 on Linux and Windows. I thought it was because there were not available as not uploaded due to the broken CI, but looks like I'm wrong.
I'm on it.

Reply:
Isn't it because `softprops/action-gh-release@v1` that uploads all zip files fails on macOS file and just stop, as a whole function, so none of zip files are uploaded yet. Even if the *Deploy articfacts* action continue with the `continue-on-error: true` ?

Reply:
Ok, looks like the `fail_on_unmatched_files: false` key for `softprops/action-gh-release@v1` is actually what we are looking for.

Reply:
you are right 👍